README.md

```diff
@@ -0,0 +1,18 @@
+# CascadeBot
+
+CascadeBot runs on JDK 11 to have the latest features available to provide the best user experience.
+
+### Project Owners
+
+Lots of talented developers have put work into CascadeBot, although the three main owners have put incredible amounts of work into CascadeBot, getting it to the stage it is now. Here's some more information about them:
+
+ - [BinaryOverload](https://github.com/binaryoverload): BinaryOverload#2382 
+ - [mrjvs](https://github.com/mrjvs): mrjvs#2165 
+ - [weeryan17](https://github.com/weeryan17) weeryan17#1258
```

Comment:
forgot a `:` here. we need to be consistent

Reply:
Oof that's my bad

Reply:
*cough* ~~@binaryoverload~~ *cough*